server/lib/middleware.js

```diff
@@ -79,7 +82,22 @@ module.exports = function middlewareConstructor(config) {
 
       // Decrypt oauth token
       req.user = req.session.user;
-      req.user.token = cryptr.decrypt(req.session.token);
+      let token = req.user.token = cryptr.decrypt(req.session.token);
```

Comment:
can we change this to "we expect an X character long, hexadecimal character string"? And directly related to that, if we know the exact length we expect, we can assert on token length as well